src/pages/Home.jsx

```diff
@@ -6,12 +6,12 @@ function Home () {
     <div className="p-6 bg-white bg-opacity-60 bg-[url('/public/lawncare-bg.jpeg')] bg-no-repeat bg-cover">
       <div className="flex flex-col items-center justify-center p-6 mt-24 bg-white rounded-2xl">
         <BaseHeading level={1} text="Yorkshire Woods Ground Crew" />
-        <p className="mt-4 text-xl font-body">
+        <p className="mt-4 font-sans text-xl">
```

Comment:
issue: Since we customised the `sans` fontFamily in `tailwind.config.js`, and since we know that [Tailwind applies this to everything in the `html` element](https://tailwindcss.com/docs/font-family#customizing-the-default-font), we should not need to add `font-sans` class to anything - we should now get it for free!